src/session.ts

```diff
@@ -217,10 +211,13 @@ export class DebugSession implements IDebugger.ISession {
  * A namespace for `DebugSession` statics.
  */
 export namespace DebugSession {
+  /**
+   * Instantiation options for a `DebugSession`.
+   */
   export interface IOptions {
     /**
-     * The client session used by the debug session.
+     * The session connection used by the debug session.
      */
-    client: IClientSession | Session.ISession;
+    sessionConnection: Session.ISessionConnection;
```

Comment:
Named `sessionConnection` for now to be explicit about what it is.

We could also rename it to just `connection` for succinctness.

Reply:
I am always a fan of succinctness.

Reply:
:+1: